requirements.txt

```diff
@@ -1,13 +1,14 @@
 # pypi requirements
+accelerate>=0.12
 datasets>=1.11.0
+diffusers>=0.2.4
 joblib>=1.1.0
 keras==2.3.1
-torchmetrics<0.7
 keybert==0.2.0
 minio==7.0.1
 modlamp>=4.0.0
-scikit-learn<0.24.0
 numpy>=1.16.5
+protobuf<3.20
```

Comment:
Thanks for this, I used `sort-requirements` to sort them lately and I have really no clue how I ended up failing on sorting this file 😱